### PR TITLE
add optional footer to _query_table

### PR DESCRIPTION
**Pull request by redstreet**
#1089

### CODE REVIEW

**Comment by redstreet** [comment]
Can someone help make this look prettier? Ideas:
- replicate the header style, or
- make the footer bold

**Comment by redstreet** [comment]
@yagebu any objections to merging this now and prettifying later?

The footer doesn't get displayed unless the optimal footer argument is passed in, which no fava code currently does.

File: fava/templates/_query_table.html [hunk 1]
```diff
@@ -60,6 +60,15 @@
     </tr>
     {% endfor %}
   </tbody>
+  {% if footer != None %}
```
Comment (yagebu) [inline fava/templates/_query_table.html:63]: {% if footer %}

File: fava/templates/_query_table.html [hunk 2]
```diff
@@ -60,6 +60,15 @@
...
+  <thead>
```
Comment (yagebu) [inline fava/templates/_query_table.html:64]: `<tfoot>`

Comment (redstreet) [reply fava/templates/_query_table.html]: Ah!

**Comment by yagebu** [comment]
Thanks!

**Pull request merged**